### PR TITLE
feat: deno test --filter

### PR DESCRIPTION
**Pull request by ry**

### CODE REVIEW

File: cli/test_runner.rs [hunk 1]
```diff
@@ -61,15 +61,24 @@ pub fn prepare_test_modules_urls(
...
     test_file.push_str(&format!("import \"{}\";\n", module.to_string()));
   }
 
-  let run_tests_cmd =
-    format!("Deno.runTests({{ failFast: {} }});\n", fail_fast);
+  let options = if let Some(filter) = filter {
+    json!({ "failFast": fail_fast, "filter": filter })
+  } else {
+    json!({ "failFast": fail_fast })
+  };
```
Comment (bartlomieju) [inline cli/test_runner.rs:79]: TIOLI: `skip` argument could be handled as well

**Comment by David-Else** [comment]
Are there some docs for this?

**Comment by ry** [comment]
@David-Else kind of.. deno help test

**Comment by David-Else** [comment]
Will have to await the next release :)

**Pull request merged**